sklearn/ensemble/tests/test_common.py

```diff
@@ -170,3 +179,24 @@ def test_ensemble_heterogeneous_estimators_all_dropped(X, y, estimator):
     estimator.set_params(lr='drop')
     with pytest.raises(ValueError, match="All estimators are dropped."):
         estimator.fit(X, y)
+
+
+@pytest.mark.parametrize(
+     "meta_est, est, X, y",
+     [(StackingClassifier, LogisticRegression,
+       X, y),
+      (StackingRegressor, LinearRegression,
+       X_r, y_r),
+      (VotingClassifier, LogisticRegression,
+       X, y),
+      (VotingRegressor, LinearRegression,
+       X_r, y_r)]
+ )
+def test_support_missing_values(meta_est, est, X, y):
+    # introduce some missing values in X
```

Comment:
```suggestion
def test_heterogeneous_ensemble_support_missing_values(Ensemble, Estimator, X, y):
    # check that Voting and Stacking predictor delegate the missing values
    # validation to the underlying estimator.
```

Reply:
you can add that this is a smoke test

Reply:
We should as well add to all test

```python
FIXME: we should move this test in `estimator_checks` once we are able
to construct meta-estimator instances
```

Reply:
Even for MultiOutput and MultiClass wrappers?